build_rln.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
Mmm. Good question. Since there are no obvious alternatives, I think the root folder is the best option _for now_, since this is where other build and environment scripts are.